### ISSUE TITLE
Name property for '-' Button is not defined in 'Preferences' window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#372]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to 'Tools' menu item present in menu bar and Expand it
3. Navigate to 'Preferences' menu item and select it
4. Navigate to 'general' tab item and select it
5. Navigate to '-'  button and Observe Inspect property 

### Expected Behavior
Name property for '-' button should be defined properly((For e.g. Decrease font size)

### Actual Behavior
Name property for '-' button is not defined and currently is Empty
**Narrator Behavior:** Narrator is announcing as 'Button'
**NVDA behavior:** NVDA is announcing as 'Button'

### User Impact
User who rely on screen reader will not be able to know the name and purpose of button 

### Screenshots
![-button_name_property_inspect](https://user-images.githubusercontent.com/22956130/40725875-224ea23e-6442-11e8-81fe-d6581e014621.png)
![-button_name_property_narrator](https://user-images.githubusercontent.com/22956130/40725887-278db03c-6442-11e8-9e03-3e70d855101d.png)
![-button_name_property_nvda](https://user-images.githubusercontent.com/22956130/40725890-2a07cd52-6442-11e8-89dd-b484296bd021.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

### Log File

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
